config/locales/en.yml

```diff
@@ -138,6 +138,10 @@ en:
               blank: "can't be blank"
             claim_ids:
               blank: "can't be blank"
+        claim/advocate_interim_claim:
```

Comment:
As discussed, despite not being a rails convention for handling error message translation I think we should adhere to it for claim errors.